src/styles/variables.scss

```diff
@@ -13,9 +13,71 @@ $select-cyan: #83dadd;
 $error-red: #f34d4d;
 
 // "Theme" Colours - specifically designed to allow for hot-swapping later
-
 $theme-light: $light-purple;
 $theme-dark: $dark-purple;
 $theme-tertiary: $gray-purple;
 $theme-highlight: $select-cyan;
-$theme-error: $error-red;
\ No newline at end of file
+$theme-error: $error-red;
+
+//Theme variables
+/* To change colors according to theme:
+ex)
+    .class{ 
+        @include themify($themes) {
+            background-color: themed("backgroundColor");
+        }
+    }
+*/
+
+$themes: (
+  light: (
+    backgroundColor: white !important,
```

Comment:
In particular this is the color I think that should be different - I've tried `#ccc`, `#DDDCDF` and I think they offer more visibility but look less clean.

Reply:
Yeah i completely agree! White was just easy for me to use as a placeholder, but definitely a little bit harsh? Same for the gutter font color, I think we could mute everything down a bit for the palette

Reply:
Do you have any suggestions on which colors you think look the best! I feel like you've got a good eye for design 😮